### PR TITLE
Turns on `enableFeatureNewHomeHero` in prod

### LINKED ISSUE #6981: Turn on enableFeatureNewHomeHero in prod
dependency: 

need to make sure guids/addons exist on stage too (currently they don't)

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #6981

### CODE REVIEW

**Comment by rebmullin** [comment]
Hey thanks so much for doing this @championshuttler. 

We not quite ready for stage yet but we can use this when we are:)

There are a few dependencies we need to address first:
all the add-ons used on the guides page should be available on stage (this is noted in issue). I think we are still missing ~ 1 which I need to look into.

Also, I believe there are few outstanding issues that need to be addressed first:
- #7092
- #7058 
- I believe there may be a few more issues on UX and changes to content so we are just waiting on these too.

**Comment by championshuttler** [comment]
> hey @championshuttler, this will actually need to be updated for prod now (for our next push date on 12/13).

Hey @rebmullin Made the changes :)

File: config/prod.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 // The following configuration values will be applied to production only.
 // However, the default config is typically used to configure production.
 module.exports = {
+
+    enableFeatureHomeHeroGuides: true,
```
[the remark this answers is not in the document]
Comment (championshuttler) [reply config/prod.js]: > I think you can just update the config/default file:
> 
> `enableFeatureHomeHeroGuides: true,`

Hi @rebmullin thanks for the suggestion, I made the changes and set the `enableFeatureHomeHeroGuides` to true in `config/default.js`

> I think we still want to keep the flag for now but turn it on everywhere (with this line). I think we can remove the flag in a later follow up issue when we know all is okay

Sorry but i did not get it , do you mean that to keep the flag in `stage` as well? If no can you please help me with here https://travis-ci.org/mozilla/addons-frontend/jobs/461927125#L691 , its expecting `length` to be `1` but we are getting length in it :disappointed:

**Comment by rebmullin** [comment]
hey @championshuttler,

thanks for updating the default config. We can also remove these specific flags listed in some of the other files:

- `config/dev-amo.js`
- `config/development-amo.js`

You shouldn't need to do anything specifically for stage. The update to the default will enable this setting in all envs.

**Comment by championshuttler** [comment]
> hey @championshuttler, thanks for updating this:) Can you also fix the test?
> 
> I think you can actually just remove the following test here:
> 
> ```diff
> --- a/tests/unit/amo/pages/TestHome.js
> +++ b/tests/unit/amo/pages/TestHome.js
> @@ -70,12 +70,6 @@ describe(__filename, () => {
>      return shallowUntilTarget(<Home {...allProps} />, HomeBase);
>    }
>  
> -  it('renders a carousel', () => {
> -    const root = render();
> -
> -    expect(root.find(HomeHeroBanner)).toHaveLength(1);
> -  });
> -
> ```
I was thinking about removing the tests today, was about to confirm about it with you tonight on IRC because i thought its not good to remove the tests , thanks for confirming before :smile:

**Comment by rebmullin** [comment]
thanks @championshuttler :)

**Comment by kumar303** [comment]
Thanks for your contribution, @championshuttler, but please ask before implementing something that doesn't already have the [`contrib: welcome`](https://github.com/mozilla/addons-frontend/labels/contrib%3A%20welcome) label. This type of issue was a trivial patch, indeed, but the timing of it was dependent on a lot of other things. That makes it not well suited to contributors.

**Pull request merged**